constraint/pkg/client/client.go

```diff
@@ -553,9 +555,14 @@ func (c *Client) Review(ctx context.Context, obj interface{}, opts ...drivers.Qu
 			continue
 		}
 
+		// edge case: .Milliseconds() can be rounded to 0 so take the nanos and convert to millis
+		resp.EvaluationLatency = float64(time.Since(reviewStart).Nanoseconds()) / 1000000
+
 		for _, autorejection := range autorejections[target] {
 			resp.AddResult(autorejection.ToResult())
 		}
+		// since autorejections don't get evaluated, adjust the batchSize to include those
+		resp.BatchSize += uint(len(autorejections))
```

Comment:
At first, I didn't add this but it makes sense to add the auto rejections to the `BatchSize` for the following reasons:

- accurately represent the `Review` objects that were actually under the `.Review()` call